sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntity.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    [CodeGenModel("HealthcareEntity")]
+    public partial class HealthcareEntity
+    {
+        /// <summary>
+        /// IsNegated
+        /// </summary>
+        [CodeGenMember("IsNegated")]
+        public bool IsNegated { get; }
```

Comment:
why the need of this customization?

Reply:
Still showing

Reply:
to make it public we need to add the docstring in customization.